UnitsNet/Scripts/Include-GenerateQuantitySourceCode.ps1

```diff
@@ -197,6 +198,33 @@ namespace UnitsNet
             return new $quantityName($func);
         }
 
+        /// <summary>
```

Comment:
Other than that, I don't see any traces of problems with the renamed files and folders, although I'll take a closer look when the AppVeyor build goes through.